curator.cabal

```diff
@@ -48,11 +46,11 @@ library
   default-language: Haskell2010
 
 executable casa-curator
-  main-is: casa-curator.hs
```

Comment:
I had some issues both with `build` and `ghci`, iirc because the binaries were in the same folder and app/curator.hs vs src/Curator.hs. Perhaps there is another way to work around this?

Reply:
I like this change regardless.